### PR TITLE
Add_Staff

### PR DESCRIPTION
**Pull request by AVI5HEK**

### CODE REVIEW

File: app/src/main/java/com/framgia/hrm/activity/Addstaff.java [hunk 1]
```diff
@@ -26,6 +53,159 @@ public void onClick(View view) {
...
+            @Override
+            public void onClick(View v) {
+                setDatefield();
+            }
+        });
+        mSactivity.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                switch (position) {
+                    case 0:
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/activity/Addstaff.java]: should not hardcode 0, 1. Please make it clear by constant name

File: app/src/main/java/com/framgia/hrm/activity/Addstaff.java [hunk 2]
```diff
@@ -26,6 +53,159 @@ public void onClick(View view) {
...
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
+        mSstatus.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                switch (position) {
+                    case 0:
+                        mStatus = 1;
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/activity/Addstaff.java]: same as above

File: app/src/main/java/com/framgia/hrm/activity/Addstaff.java [hunk 3]
```diff
@@ -26,6 +53,159 @@ public void onClick(View view) {
...
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
+        mSposition.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                switch (position) {
+                    case 0:
+                        mPosition = 1;
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/activity/Addstaff.java]: same!

File: app/src/main/java/com/framgia/hrm/activity/MainActivity.java [hunk 4]
```diff
@@ -46,51 +46,60 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        /*PREF_STATE = getApplicationContext().getSharedPreferences("MyPref",
                 MODE_PRIVATE);
-        int pref_State = PREF_STATE.getInt("state",0);
+        int pref_State = PREF_STATE.getInt("state",0);*/
         drawerFragment = (FragmentDrawer) getSupportFragmentManager().findFragmentById(R.id.fragment_navigation_drawer);
         drawerFragment.setUp(R.id.fragment_navigation_drawer, (DrawerLayout) findViewById(R.id.drawer_layout), mToolbar);
         drawerFragment.setDrawerListener(this);
         displayView(0);
-
-        if(pref_State!=3) {
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/activity/MainActivity.java]: 3 = ???, could you make any paramater like this clearer???

File: app/src/main/java/com/framgia/hrm/activity/MainActivity.java [hunk 5]
```diff
@@ -46,51 +46,60 @@ protected void onCreate(Bundle savedInstanceState) {
...
-            mdDatabaseHelper.createPosition(new Position("DEPUTY MANAGER"));
-            mdDatabaseHelper.createPosition(new Position("SR. EXECUTIVE"));
-            mdDatabaseHelper.createPosition(new Position("EXECUTIVE"));
-            mdDatabaseHelper.createActivity(new Activity("ACTIVE"));
-            mdDatabaseHelper.createActivity(new Activity("INACTIVE"));
-
+        //if(pref_State!=3)
+        if (!mDatabaseHelper.departmentTableExists() && !mDatabaseHelper.statusTableExists() &&
+                !mDatabaseHelper.positionTableExists() && !mDatabaseHelper.activityTableExists()) {
+            mDatabaseHelper.createDepartment(new Department("HRM & ADMIN"));
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/activity/MainActivity.java]: how's about building a method to create a list of department,....???

Comment (AVI5HEK) [inline app/src/main/java/com/framgia/hrm/activity/MainActivity.java]: Like adding the names of departments to an ArrayList and then pass it as an argument to a method where it creates all the departments by taking the names from that ArrayList? I can build that on Monday first thing in the morning. :)

**Pull request closed without merging**